extensions/amp-sidebar/1.0/storybook/Basic.amp.js

```diff
@@ -65,14 +56,34 @@ export const _default = () => {
   );
 };
 
-export const toolbar = () => {
-  const sideConfigurations = ['left', 'right', undefined];
-  const side = select('side', sideConfigurations, sideConfigurations[0]);
-  const foregroundColor = color('color');
-  const backgroundColor = color('background');
-  const backdropColor = color('backdrop color');
-  const toolbarMedia = text('toolbar media', '(max-width: 500px)');
+_default.argTypes = {
+  side: {
+    name: 'side',
+    defaultValue: 'left',
+    options: ['left', 'right', undefined],
```

Comment:
Does this need to be `undefined` or can be `none`?

Reply:
In this case it needs to be `undefined` so that the `size` attr is not set. Setting `size='none'` does not trigger the correct behavior.

Reply:
nit: can u add this explanation as a comment so its not confusing why `undefined` is one of the options.